### PR TITLE
Mutable default value 1817

### PR DESCRIPTION
**Pull request by emanlove**
Fixes issues with mutable default values.

### CODE REVIEW

File: src/SeleniumLibrary/keywords/browsermanagement.py [hunk 1]
```diff
@@ -340,7 +340,7 @@ def _make_new_browser(
 
     @keyword
     def create_webdriver(
-        self, driver_name: str, alias: Optional[str] = None, kwargs={}, **init_kwargs
+        self, driver_name: str, alias: Optional[str] = None, kwargs=None, **init_kwargs
```
Comment (Snooz82) [inline src/SeleniumLibrary/keywords/browsermanagement.py]: You should add the type hin here:
`kwargs: Optional[Dict] = None`

File: src/SeleniumLibrary/keywords/browsermanagement.py [hunk 2]
```diff
@@ -371,6 +371,8 @@ def create_webdriver(
         `Close All Browsers` keyword is used. See `Switch Browser` for an
         example.
         """
+        if kwargs is None:
+            kwargs = {}
         if not isinstance(kwargs, dict):
             raise RuntimeError("kwargs must be a dictionary.")
```
Comment (Snooz82) [inline src/SeleniumLibrary/keywords/browsermanagement.py:368]: These two line you could removed. 
Robot Framework will fail, if you give anything that can not be converted to dict.
At least if we are not covering every bad things people could do from Python.

**Review by Snooz82 — CHANGES_REQUESTED** [review]
Two comments. 
The type hint is important. Otherwise it will use the default as type hint.

**Review by yuriverweij — COMMENTED** [review]
Agree with Snooz82, nothing else to add

File: src/SeleniumLibrary/keywords/browsermanagement.py (hunk 2 above)
Comment (emanlove) [reply src/SeleniumLibrary/keywords/browsermanagement.py:368]: For these I would prefer to keep them there. Relying on another piece code, even a single entry point type path, is something I prefer not to do.

**Review by Snooz82 — APPROVED** [review]
looks good to me

**Comment by emanlove** [comment]
The failing rf v6.1.1 failing test is unrelated so proceeding with the merge of this fix ..

**Pull request merged**